app/models/manageiq/providers/ansible_tower/automation_manager/amazon_credential.rb

```diff
@@ -1,2 +1,10 @@
 class ManageIQ::Providers::AnsibleTower::AutomationManager::AmazonCredential < ManageIQ::Providers::AnsibleTower::AutomationManager::CloudCredential
+  EXTRA_ATTRIBUTES = {
+    :security_token => {
+      :type       => :password,
+      :label      => _('Security Token'),
```

Comment:
As things are, `_()` will not work here. Since `EXTRA_ATTRIBUTES` is a constant and will be evaluated
too early, with `_()` this would never be translated into other locales (i.e. the string inside will always
be in English.

What you need to do here (and same with all the other cases of this below) is replace `_()` with `N_()`